checkST2110.js

```diff
@@ -1532,33 +1701,35 @@ const allSections = (sdp, params) => {
     // Load tests based on encoding name
     if (mtParams[0]._encodingName == 'jxsv') {
       sections = [
-        section_10_62, section_10_74, section_10_81, section_10_82, section_10_83,
+        section_10_62, section_10_74, section_10_81, section_10_82, section_10_83, section_10_2022_87,
         section_21_81, section_21_82,
-        section_22_60, section_22_72, section_22_73, section_22_74];
+        section_22_60, section_22_72, section_22_73, section_22_74, section_22_2022_72];
       if (params.noCopy) {
         sections.push(no_copy_22);
       }
     } else if (mtParams[0]._encodingName == 'raw') {
       sections = [
-        section_10_62, section_10_74, section_10_81, section_10_82, section_10_83,
+        section_10_62, section_10_74, section_10_81, section_10_82, section_10_83, section_10_2022_87,
         section_20_71, section_20_72, section_20_73, section_20_74,
         section_20_75, section_20_76, section_21_81, section_21_82];
       if (params.noCopy) {
         sections.push(no_copy_20);
       }
-    }
-    else {
+    } else if (mtParams[0]._encodingName == 'smpte291') {
+      sections = [section_10_62, section_10_74, section_10_81, section_10_82,
+        section_10_83, section_10_2022_87, section_40_2023_7];
```

Comment:
Trivial quibble... Despite the long line as a result, maybe prefer this formatted consistently with the other cases, `section_10_xx` on one line and `section_40_xx` on the next?

Reply:
I missed that one - it's an artifact of Visual Studio's "helpful" auto-formatting which I've now switched off. Will fix.